hwy/ops/x86_128-inl.h

```diff
@@ -5616,6 +5634,32 @@ HWY_API Vec128<double, N> ConvertTo(Simd<double, N, 0> dd,
 #endif
 }
 
+template <size_t N>
+HWY_API Vec128<double, N> ConvertTo(HWY_MAYBE_UNUSED Simd<double, N, 0> dd,
+                                    const Vec128<uint64_t, N> v) {
+#if HWY_TARGET <= HWY_AVX3
+  return Vec128<double, N>{_mm_cvtepu64_pd(v.raw)};
+#else
+  // Based on wim's approach (https://stackoverflow.com/questions/41144668/)
+  const Repartition<uint64_t, decltype(dd)> d64;
+
+  const auto msk_lo = Set(d64, 0xFFFFFFFF);
+  const auto cnst2_32_dbl = Set(dd, 4294967296.0); /* 2^32 */
+  const auto v_lo = And(v, msk_lo); /* Extract the 32 lowest significant bits of v */
+  const auto v_hi = ShiftRight<32>(v);
+
+  using vec_type = typename std::decay<decltype(v_lo)>::type;
```

Comment:
You can also say `using VU = VFromD<decltype(d64)>;`. Would be nice to then use this VU type instead of auto for a bit more clarity/readability.